cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -836,3 +838,51 @@ func TestSimulateTransactionBumpAndRestoreFootprint(t *testing.T) {
 	assert.NoError(t, err)
 	sendSuccessfulTransaction(t, client, sourceAccount, tx)
 }
+
+func TestCLI(t *testing.T) {
+	test := NewTest(t)
+	ch := jhttp.NewChannel(test.sorobanRPCURL(), nil)
+	client := jrpc2.NewClient(ch, nil)
+
+	sourceAccount := keypair.Root(StandaloneNetworkPassphrase)
+
+	tx, err := txnbuild.NewTransaction(txnbuild.TransactionParams{
+		SourceAccount: &txnbuild.SimpleAccount{
+			AccountID: keypair.Root(StandaloneNetworkPassphrase).Address(),
+			Sequence:  1,
+		},
+		IncrementSequenceNum: false,
+		Operations: []txnbuild.Operation{&txnbuild.CreateAccount{
+			Destination: "GDIY6AQQ75WMD4W46EYB7O6UYMHOCGQHLAQGQTKHDX4J2DYQCHVCR4W4",
+			Amount:      "100000",
+		}},
+		BaseFee: txnbuild.MinBaseFee,
+		Memo:    nil,
+		Preconditions: txnbuild.Preconditions{
+			TimeBounds: txnbuild.NewInfiniteTimeout(),
+		},
+	})
+	require.NoError(t, err)
+	sendSuccessfulTransaction(t, client, sourceAccount, tx)
+	binary, lookErr := exec.LookPath("cargo")
+	if lookErr != nil {
+		panic(lookErr)
+	}
+
+	// args := []string{"cargo", "test", "--package", "soroban-test", "--test", "it", "--", "invoke_sandbox::from_go", "--exact", "--nocapture", "--ignored"}
+	args := []string{"cargo", "run", "--", "--vv", "contract", "install", "--wasm ../../../../target/wasm32-unknown-unknown/test-wasms/test_hello_world.wasm"}
+	env := os.Environ()
+	env = append(env, "SOROBAN_RPC_URL=http://localhost:8000/", "SOROBAN_NETWORK_PASSPHRASE=Standalone Network ; February 2017")
+
+	execErr := syscall.Exec(binary, args, env)
```

Comment:
Are you sure this is what you want? It will cause the test process to turn into the binary execution.

I think you want to use https://pkg.go.dev/os/exec#Cmd.Run or, even better, something like https://pkg.go.dev/gotest.tools/v3/icmd as we discussed